### ISSUE TITLE
training error? odd DIST plot

### ISSUE THREAD

**Issue by happyqiu** [kristinbranson/APT#408]
Hi,
I'm using the APT-develop branch, and found that during the training, the DIST panel didn't look right while the Loss might be okay. Besides, after the training, I tried to track another video, but I didn't get any predicted labels. Do you have any ideas about what's happening here?

Thanks!
![DIST](https://user-images.githubusercontent.com/50755486/198843584-9ff77737-707a-4d63-9b15-4023e2bdffdf.png)
![track_img](https://user-images.githubusercontent.com/50755486/198843586-f85c573b-757a-4ae2-9825-3241dc9965f9.png)

**Comment by allenleetc** [comment]
Hi @happyqiu,

Can you please share your project (.lbl) file and the movie you are trying to track? You will probably need to share a link to a cloud service (eg Google Drive) because these files will be too large to directly attach here.

When you track the video, have you looked at the log in the Tracking Monitor? There might be warnings or other messages printed there. If this log is available and you can upload it here that might also be useful.

Thanks!

**Comment by happyqiu** [comment]
Thanks for your reply.
Please find the link below. (video 20 is for tracking.)
https://drive.google.com/drive/folders/1gu0R8yqWSzz7APH54vOyIpgPIAOqHa_C?usp=sharing
Sorry that I don't have the log messages from the tracking monitor.

**Comment by allenleetc** [comment]
@happyqiu 

Strange, if I try tracking with your trained tracker, I get predictions but they are 'garbage' (all in the upper-left corner). Maybe that is why you don't see them?

However, if I retrain, my loss/dist plots look normal and the tracking looks good.

Nothing jumps out yet -- maybe if it's not difficult, try doing a fresh retrain to see if anything changes? (Please save the training log just in case.) So far your training data looks normal so I wonder if it could be something in your environment/platform.

**Comment by happyqiu** [comment]
That happens...
When I tried this .lbl file with a TITAN X GPU computer, it worked alright.
But with my own computer (NVIDIA RTX A5000), this problem occurred. Does
the GPU matter that much? All other installations are the same on these two
computers.

**Comment by allenleetc** [comment]
It looks like a compatibility issue with the A5000 may be possible. In **develop** we are on tf1.15 and see eg 

https://discuss.tensorflow.org/t/tensorflow-and-cuda-support-for-latest-nvida-a5000-ampere-gpu/3886
https://embea.de/blog/?p=114

@mkabra could @happyqiu have Ampere compatibility issues even if they switch to the **multianimal** branch? One of these links seems to suggest that tf2.4 is required. 

In general the specific GPU can potentially matter as in eg https://github.com/kristinbranson/APT/issues/365.

**Comment by mkabra** [comment]
Yes, even on multi-animal branch there will be compatibility issues with Ampere architectures.

@happyqiu, we are looking at updating the software to work with the newer architectures, but it could take a few months to do that.

Mayank
________________________________
